gallery/visualtests/nxList.js

```diff
@@ -6,22 +6,21 @@
  */
 const { hoverTest, simpleTest } = require('./testUtils');
 
-describe('nx-list', function() {
+describe('NxList', function() {
```

Comment:
I don't really see a need to change the visual tests.  The `nx-list` styles page demonstrates every different way it can look, and that is already what was being tested.  The `NxList` component is just a wrapper around those styles and so doesn't have any additional/different visuals to test.

Reply:
> I don't really see a need to change the visual tests. The `nx-list` styles page demonstrates every different way it can look, and that is already what was being tested. The `NxList` component is just a wrapper around those styles and so doesn't have any additional/different visuals to test.

Visual tests have been reverted to their original.
3ad8fff9bb353f592fd4fc440cc0387364445aba